datastore/db/metadata/inspection/__init__.py

```diff
@@ -45,6 +45,12 @@ class Value(BaseModel):
     name: Optional[str] = None
     edited: Optional[bool] = False
 
+class GuaranteedAnalysis(BaseModel):
+    title: Optional[str] = None
+    titre: Optional[str] = None
```

Comment:
Here as well there's a break from the usual pattern:
```json
{
    "title": {"en": "", "fr": ""}
}
```

Though, I don't have any strong argument against this.

Reply:
I'd have to update the getter to create the title into it's own object. Hopefully the JSON function of PostgreSQL wont bother me too much.

Reply:
Actually I strongly suggest we make these changes. This will be good for modifiability.